cns/logger/log.go

```diff
@@ -22,113 +24,90 @@ var (
 type CNSLogger struct {
 	logger               *log.Logger
 	th                   aitelemetry.TelemetryHandle
-	Orchestrator         string
-	NodeID               string
 	DisableTraceLogging  bool
 	DisableMetricLogging bool
 	DisableEventLogging  bool
-}
 
-// Initialize CNS Logger
-func InitLogger(fileName string, logLevel, logTarget int, logDir string) {
-	Log = &CNSLogger{
```

Comment:
❤️

Reply:
ditto, but maybe fixing the logger should be a separate set of changes (that go first, if necessary)?